vars/piperStageWrapper.groovy

```diff
@@ -80,7 +80,7 @@ private void executeStage(script, originalStage, stageName, config, utils, telem
         def body = originalStage
 
         // First, check if a global extension exists via a dedicated repository
-        if (globalExtensions) {
+        if ((env.PIPER_DISABLE_EXTENSIONS==null || globalExtensions) && (!Boolean.valueOf(script.env.PIPER_DISABLE_EXTENSIONS) && globalExtensions)) {
```

Comment:
one `env.PIPER_DISABLE_EXTENSIONS` is prefixed with `script`, the other one not. This should be aligned.